lib/resty/aes.lua

```diff
@@ -137,6 +139,11 @@ function _M.new(self, key, salt, _cipher, _hash, hash_rounds, iv_len)
     local gen_key = ffi_new("unsigned char[?]",_cipherLength)
     local gen_iv = ffi_new("unsigned char[?]",_cipherLength)
     iv_len = iv_len or _cipherLength
+    padding = padding or 1
```

Comment:
seems a boolean value is more meaningful?

Reply:
https://github.com/openssl/openssl/blob/master/include/openssl/evp.h#L438-L443

There are 5 modes of padding

Reply:
ok